src/EditorFeatures/Test2/Rename/RenameNonRenameableSymbols.vb

```diff
@@ -482,5 +482,165 @@ End Class
                 AssertTokenNotRenamable(workspace)
             End Using
         End Sub
+
+#Region "Rename In Tuples"
+
+        <WorkItem(10898, "https://github.com/dotnet/roslyn/issues/10898")>
+        <WorkItem(10567, "https://github.com/dotnet/roslyn/issues/10567")>
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Rename)>
```

Comment:
Here's the fully qualified name: `Microsoft.CodeAnalysis.Test.Utilities.Traits.Features.Tuples`.